go.mod

```diff
@@ -3,6 +3,8 @@ module github.com/minio/mc
 go 1.13
 
 require (
+	github.com/aliyun/aliyun-oss-go-sdk v0.0.0-20190307165228-86c17b95fcd5 // indirect
```

Comment:
aliyun dependency is deleted from server, we don't need this anymore have you run `go mod tidy?` @vadmeste

Reply:
> aliyun dependency is deleted from server, we don't need this anymore have you run `go mod tidy?` @vadmeste

Done